src/tests/mocks/FormProps.ts

```diff
@@ -0,0 +1,165 @@
+import type { UiSchema } from '@rjsf/core';
+import VernaJSONSchemaType from '../../types/rjsf';
+import SelectWidget from './SelectWidget';
+
```

Comment:
You could rename this file `factories.ts`. We could add in this file all factories we need to fake data for our tests.
You can take a look in marsha project to see how to make customisable factories. (https://github.com/openfun/marsha/blob/master/src/frontend/utils/tests/factories.ts)